### PR TITLE
Algorithms and Flowcharts In C Programming Language

### LINKED ISSUE #2339: Algorithms and Flowcharts In C Programming Language
### Description

- What is an Algorithm?
- Characteristics of an Algorithm?
- How to design an Algorithm?
- What is a Flowchart?
- What is DFD(Data Flow Diagram)?
- Some examples of Flowcharts

### Domain

C/CPP

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by ikavyajain**
<hr>

## Description 📜
I have prepared a documentation on:

Algorithms and Flowcharts In C Programming Language

- [x] What is an Algorithm?
- [x] Characteristics of an Algorithm?
- [x] How to design an Algorithm?
- [x] What is a Flowchart?
- [x] What is DFD(Data Flow Diagram)?
- [x] Some examples of Flowcharts

Hope you learn and applyyy! 💯 

Fixes #2339

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] C/CPP

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by vasu-1** [comment]
Great ! Now REmove that line from readme

**Comment by vasu-1** [comment]
Listen again problem

**Comment by vasu-1** [comment]
Delete that input output operations.md file and remove the line from that readme file

**Comment by vasu-1** [comment]
oyyyyyyyyyyyyyyyyyyyyyyy don't create that one again

**Comment by ikavyajain** [comment]
🥺🙄

**Comment by vasu-1** [comment]
see it again happened because of you are working on same C_CPP branch.
i told you that work on different brach for different Issues !

**Comment by vasu-1** [comment]
Please Do This Step !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
- Delete input_Output_Operation_In_C.md
- and remove the line from readme

**Comment by vasu-1** [comment]
And don't do anything...! after this

**Comment by ikavyajain** [comment]
Deleted input output and readme

**Comment by ikavyajain** [comment]
Now tell me what to do
After this

**Comment by ikavyajain** [comment]
The conflict in algorithms branch is resolved but, files changed are
appearing as 3

**Comment by vasu-1** [comment]
> The conflict in algorithms branch is resolved but, files changed are appearing as 3

wait please

File: C_CPP/Fundamentals/Input & Output Operations/readme.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
-
 - [Insertion and Extraction operator in C++](./Insertion&Extraction_Operator_in_C++.md)
+
```
Comment (vasu-1) [inline C_CPP/Fundamentals/Input & Output Operations/readme.md:2]:
```suggestion

- [Insertion and Extraction operator in C++](./Insertion&Extraction_Operator_in_C++.md)

```

**Review by vasu-1 — CHANGES_REQUESTED** [review]
just make a space over there

**Comment by vasu-1** [comment]
move that link to second line
then commit your changes

**Comment by ikavyajain** [comment]
These are two different issues

>

**Comment by vasu-1** [comment]
> These are two different issues
> […](#)

mene kaha woh try karo fir dekhenge !!!

File: C_CPP/Fundamentals/Input & Output Operations/readme.md [hunk 2]
```diff
@@ -1,2 +1,4 @@
 
+
 - [Insertion and Extraction operator in C++](./Insertion&Extraction_Operator_in_C++.md)
```
Comment (vasu-1) [inline C_CPP/Fundamentals/Input & Output Operations/readme.md:3]:
```suggestion

- [Insertion and Extraction operator in C++](./Insertion&Extraction_Operator_in_C++.md)
```

**Review by vasu-1 — CHANGES_REQUESTED** [review]
again changes !!!
you have put two lines ...!
i have told you please put only one line !

**Comment by vasu-1** [comment]
image
readme must look like this

**Comment by ikavyajain** [comment]
done!

**Comment by vasu-1** [comment]
It will be okk !
don't worry !
but again your content is from GFG yes or no?

**Comment by ikavyajain** [comment]
noo

**Comment by ikavyajain** [comment]
i have myself written this after reading my notes!

**Comment by vasu-1** [comment]
Ok And that image ?
if you have taken that image from somewhere then put credit !

**Comment by ikavyajain** [comment]
i cannot create another PR from the same branch unless this gets merged. is it so?

**Comment by vasu-1** [comment]
> i cannot create another PR from the same branch unless this gets merged. is it so?

you can create but ! work on another branch , not on C_CPP

**Comment by ikavyajain** [comment]
For the image i have copied the url from google. since, i won't be able to draw flowcharts in md

**Comment by vasu-1** [comment]
Another means create new one

**Comment by vasu-1** [comment]
Another means create new one

**Comment by ikavyajain** [comment]
hope you believe me for this. I have written the entire stuff myself! :)

**Comment by ikavyajain** [comment]
okay

**Comment by vasu-1** [comment]
> For the image i have copied the url from google. since, i won't be able to draw flowcharts in md

Ok NP

**Comment by vasu-1** [comment]
areeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee don't create again

**Comment by vasu-1** [comment]
you have created again on C_CPP branch

**Comment by ikavyajain** [comment]
muje ek repo bnani hai i have an issue on the same! kese kru phir?

**Comment by ikavyajain** [comment]
wait deleting

**Comment by ikavyajain** [comment]
okay

**Comment by vasu-1** [comment]
C_CPP ko abhi aaram karne do !!!!

**Review by vasu-1 — APPROVED** [review]
Finally after 50+ conversations ..
good to go for merge !!!! @ikavyajain  good work ! 

@Suvraneel abhi isko merge kar do bhai !!!!

**Comment by ikavyajain** [comment]
Thanks😊

**Pull request closed without merging**